src/analysis/graphspace.py

```diff
@@ -3,15 +3,18 @@
 import sys
 
 import networkx as nx
+import pandas as pd
 from graphspace_python.api.client import GraphSpace
 from graphspace_python.graphs.classes.gsgraph import GSGraph
 
 
-def write_json(graph_file,out_graph,out_style,directed=False) -> None:
+# remove all the the directed = bool in the function names
```

Comment:
```suggestion
```
I don't think we need to comment on this. Once they are gone, the comment won't have much context.